x/ibc/light-clients/99-ostracon/types/proposal_handle.go

```diff
@@ -14,12 +14,12 @@ import (
 // CheckSubstituteAndUpdateState will try to update the client with the state of the
 // substitute if and only if the proposal passes and one of the following conditions are
 // satisfied:
-//	1) AllowUpdateAfterMisbehaviour and Status() == Frozen
-// 	2) AllowUpdateAfterExpiry=true and Status() == Expired
+//  1. AllowUpdateAfterMisbehaviour and Status() == Frozen
```

Comment:
this

Reply:
oh I got it and this modification is auto-generated by goalngci-lint autofix (every changes related to lint are modified by go fmt default autofix). So you wanna keep this format instead of follow golangci-lint auto fix?

Reply:
I see, it was by goalngci-lint. But as far as IBC goes, there were a lot of comment fixes, so I thought it would be too much to include this irrelevant fix in this PR. What do you think?

Reply:
yeah it might be confused. so we have two options.
1. revert 96c2c9e7d2ce1d0873e42317020b3abbc93116f8 and lint only for modified files again
2. keep those lint changes for consistency and review PR commit by commit (before 96c2c9e7d2ce1d0873e42317020b3abbc93116f8)

which option is better you think?

Reply:
I like option 1. I checked the [commit](https://github.com/line/lbm-sdk/commit/96c2c9e7d2ce1d0873e42317020b3abbc93116f8) and thought that these many changes would be better split into other PRs.